src/ServiceControlInstaller.Engine/Instances/ServiceControlCoreTransports.cs

```diff
@@ -46,7 +46,7 @@ public class ServiceControlCoreTransports
                 AvailableInSCMU = true,
                 Matches = name => name.Equals(TransportNames.AzureStorageQueue, StringComparison.OrdinalIgnoreCase)
                                   || name.Equals("ServiceControl.Transports.ASQ.ASQTransportCustomization, ServiceControl.Transports.ASQ", StringComparison.OrdinalIgnoreCase)
-                                  || name.Equals("NServiceBus.AzureStorageQueueTransport, NServiceBus.Azure.Transports.WindowsAzureStorageQueues", StringComparison.OrdinalIgnoreCase)
+                                  || name.Equals("NServiceBus.AzureStorageQueueTransport, NServiceBus.Transport.AzureStorageQueues", StringComparison.OrdinalIgnoreCase)
```

Comment:
Same here. This shouldn't change, but a new addition should have been made

Reply:
Same https://github.com/Particular/ServiceControl/pull/2301#discussion_r531116262

Reply:
Correct. No change should be made here at all.